channels/alpha

```diff
@@ -130,28 +130,32 @@ spec:
     recommendedVersion: 1.11.10
     requiredVersion: 1.11.10
   kopsVersions:
+  - range: ">=1.24.0-alpha.1"
+    recommendedVersion: "1.23.2"
+    #requiredVersion: 1.23.0
+    kubernetesVersion: 1.23.8
```

Comment:
Also, a section for 1.24 should be added above, in the `kubernetesVersions` section.
```suggestion
    kubernetesVersion: 1.24.2
```

Reply:
Thanks @hakman!
Fixing.
As for the `1.24` section in `kubernetesVersions` - does the below look right? I mostly wasn't sure about the `requiredVersion` part, since technically `1.24.0` hasn't been released yet.
Do we just point to the most recent 1.24 release ([v1.24.0-beta.2](https://github.com/kubernetes/kops/releases/tag/v1.24.0-beta.2))? Or should it point to `1.23.0` for now?

Reply:
Yup, that looks good. Required version refers to k8s version I think.

Reply:
Of course it does, lol 🤦🏻 
Thanks for clarifying!